pandas/core/series.py

```diff
@@ -3833,9 +3834,94 @@ def argsort(
         )
         return res.__finalize__(self, method="argsort")
 
-    def nlargest(
-        self, n: int = 5, keep: Literal["first", "last", "all"] = "first"
+    def nsorted(
+        self,
+        n: int,
+        ascending: bool,
+        keep: NsmallestNlargestKeep = "first",
     ) -> Series:
+        """
+        Return the top `n` elements, sorted in ascending or descending order.
+
+        Parameters
+        ----------
+        n : int, default 5
+            Return this many descending sorted values.
+        ascending : bool
+            If True, return the smallest `n` elements. If False, return
+            largest `n` elements.
+        keep : {'first', 'last', 'all'}, default 'first'
+            When there are duplicate values that cannot all fit in a
+            Series of `n` elements:
+
+            - ``first`` : return the first `n` occurrences in order
+              of appearance.
+            - ``last`` : return the last `n` occurrences in reverse
+              order of appearance.
+            - ``all`` : keep all occurrences. This can result in a Series of
+              size larger than `n`.
+
+        Returns
+        -------
+        Series
+            The `n` top values in the Series.
+
+        See Also
```

Comment:
Since nsorted got added to the series API docs, I believe it requires its proper docstring as well. There is a lot of redundancy with nlargest and nsmallest. Let me know if it should be done differently.